app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -257,16 +258,18 @@ fun UserCard(context: Context) {
 
 @Composable
 fun AssociationProfileEvents(context: Context) {
-  val eventViewModel: EventViewModel = viewModel(factory = EventViewModel.Factory)
+  val userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory)
 
   Column(
       modifier = Modifier.padding(horizontal = 28.dp),
       horizontalAlignment = Alignment.CenterHorizontally) {
         Box(modifier = Modifier.testTag("AssociationEventCard")) {
           EventCard(
               event =
-                  Event(organisers = MockReferenceList(), taggedAssociations = MockReferenceList()),
-              viewModel = eventViewModel)
+                  Event(
+                      organisers = Association.emptyFirestoreReferenceList(),
+                      taggedAssociations = Association.emptyFirestoreReferenceList()),
+              userViewModel = userViewModel)
```

Comment:
Why use a "User" view model to fetch "Events"?

Reply:
Because the saved events is something that inherently belongs to the User. You can see Oskar explanation above